src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const payPerDay = 40;
+  const discountAfterThreeDays = 20;
+  const discountAfterSevenDays = 50;
+
+  if (days < 3) {
+    result = days * payPerDay;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  } else if (days < 7) {
```
The first check is not necessary. If first case `(days < 3)` won't pass it will mean that days >=3